src/hooks/useCds/translate.js

```diff
@@ -0,0 +1,185 @@
+        code: "373887005"
+      },
+      {
+        text: "Atypical squamous cells of undetermined significance",
+        code: "441087007"
+      },
+      {
+        text: "Atypical squamous cells of undetermined significance, cannot exclude high grade",
+        code: "441088002"
+      },
+      {
+        text: "Low grade squamous intraepithelial lesion",
+        code: "62051000119105"
+      },
+      {
+        text: "High grade squamous intraepithelial lesion",
+        code: "62061000119107"
+      },
+      {
+        text: "High grade squamous intraepithelial lesion, moderate dysplasia",
+        code: "62061000119107"
+      },
+      {
+        text: "High grade squamous intraepithelial lesion, severe dysplasia",
+        code: "62061000119107"
+      },
+      {
+        text: "High grade squamous intraepithelial lesion, Carcinoma in situ dysplasia",
+        code: "62061000119107"
+      },
+      {
+        text: "invasive squamous cell carcinoma",
+        code: "416033009"
+      },
+      {
+        text: "Atypical glandular cells, NOS",
+        code: "441219009"
+      },
+      {
+        text: "Atypical endocervial cells, NOS",
+        code: "448406002"
+      },
+      {
+        text: "Atypical endometrial cells, NOS",
+        code: "441219009"
+      },
+      {
+        text: "Endocervical adenocarcinoma in situ",
+        code: "254890008"
+      },
+      {
+        text: "Adenocarcinoma, not otherwise specified",
+        code: "254890008"
+      },
+      {
+        text: "invasive endocervical adenocarcinoma",
+        code: "254890008"
+      },
+      {
+        text: "Endometrial adenocarcinoma",
+        code: "123845008"
+      },
+      {
+        text: "Other malignant neoplasm, see comment",
+        code: "416033009"
+      },
+      {
+        text: "High grade squamous intraepithelial lesion, cannot rule out invasion",
+        code: "373879009"
+      },
+      {
+        text: "Unable to interpret",
+        code: "126481000119106"
+      },
+      {
+        text: "Malignant cells present",
+        code: "416033009"
+      },
+      {
+        text: "Reparative/reactive changes",
+        code: "373887005"
+      }
+    ]
+  },
+  {
+    testCode: ['71431-1', '77399-4', '77400-0'],
+    testName: 'HPV',
+    map: [
+      {
+        text: "Not Detected",
+        code: "260415000"
+      },
+      {
+        text: "Negative",
+        code: "260415000"
+      },
+      {
+        text: "Detected",
+        code: "260373001"
+      },
+      {
+        text: "Postive",
+        code: "260373001"
+      }
+    ]
+  }
+]
+
+const loincMapping = [
+  {
+    oldCode: '61372-9',
+    newCode: '77399-4'
+  },
+  {
+    oldCode: '61373-7',
+    newCode: '77400-0'
+  }
+];
+
+const SCT_URL = 'http://snomed.info/sct'
+const LOINC_URL = 'http://loinc.org'
+
+
+/**
+ * Translate terminologies in Observation and DiagnosticReport
```

Comment:
The code here is fine for now, since the data for this pilot is only coming in as Observations, but we might want to either remove the reference to DiagnosticReport here, or add a "ToDo" in a comment stating that we will add mapping for DiagnosticReports.

Reply:
Thanks!